diff -Nru mm-1.0.12/Makefile.in mm-1.0.12.new/Makefile.in
--- mm-1.0.12/Makefile.in	Sat Sep 25 15:08:50 1999
+++ mm-1.0.12.new/Makefile.in	Sat Dec 18 13:00:35 1999
@@ -156,16 +156,16 @@
 	shtoolize -o shtool all
 
 install: all
-	$(SHTOOL) mkdir -f -p -m 755 $(bindir)
-	$(SHTOOL) mkdir -f -p -m 755 $(includedir)
-	$(SHTOOL) mkdir -f -p -m 755 $(libdir)
-	$(SHTOOL) mkdir -f -p -m 755 $(mandir)/man1
-	$(SHTOOL) mkdir -f -p -m 755 $(mandir)/man3
-	$(SHTOOL) install -c -m 755 mm-config $(bindir)/mm-config
-	$(SHTOOL) install -c -m 644 mm-config.1 $(mandir)/man1/mm-config.1
-	$(SHTOOL) install -c -m 644 mm.3 $(mandir)/man3/mm.3
-	$(SHTOOL) install -c -m 644 mm.h $(includedir)/mm.h
-	@$(LIBTOOL) --mode=install $(SHTOOL) install -c -m 644 libmm.la $(libdir)/libmm.la
+	$(SHTOOL) mkdir -f -p -m 755 $(DESTDIR)$(bindir)
+	$(SHTOOL) mkdir -f -p -m 755 $(DESTDIR)$(includedir)
+	$(SHTOOL) mkdir -f -p -m 755 $(DESTDIR)$(libdir)
+	$(SHTOOL) mkdir -f -p -m 755 $(DESTDIR)$(mandir)/man1
+	$(SHTOOL) mkdir -f -p -m 755 $(DESTDIR)$(mandir)/man3
+	$(SHTOOL) install -c -m 755 mm-config $(DESTDIR)$(bindir)/mm-config
+	$(SHTOOL) install -c -m 644 mm-config.1 $(DESTDIR)$(mandir)/man1/mm-config.1
+	$(SHTOOL) install -c -m 644 mm.3 $(DESTDIR)$(mandir)/man3/mm.3
+	$(SHTOOL) install -c -m 644 mm.h $(DESTDIR)$(includedir)/mm.h
+	@$(LIBTOOL) --mode=install $(SHTOOL) install -c -m 644 libmm.la $(DESTDIR)$(libdir)/libmm.la
 
 clean:
 	$(RM) mm_test mm_test.o mm_test.lo
